app/jobs/queue_solution_head_test_runs_job.rb

```diff
@@ -0,0 +1,7 @@
+class QueueSolutionHeadTestRunsJob < ApplicationJob
+  queue_as :default
```

Comment:
Were you planning to create a separate queue for this?

Reply:
Hmmm, not considered it. I copied the existing one (`MarkSolutionsAsOutOfDateInIndexJob`)